lib/AST/ClangTypeConverter.cpp

```diff
@@ -850,6 +852,18 @@ clang::QualType ClangTypeConverter::convert(Type type) {
                             1);
         return ctx.getObjCObjectPointerType(clangType);
       }
+    } else {
+      auto it = ReversedExportMapBackwards.find(decl);
+      // This can happen, when the String type is exported to C++ (as
+      // swift::String) which is later imported back to Swift as String.
+      if (it != ReversedExportMapBackwards.end()) {
+        const auto *clangDecl = it->getSecond();
+        const auto *dc = clangDecl->getDeclContext();
+        if (dc->isNamespace() &&
+            cast<clang::NamespaceDecl>(dc)->getName() == "swift")
```

Comment:
Can we name the "swift" constant here? That same string is used in so many different ways, and while I can tell that it's a namespace name from here, I wasn't clear whether this is called "swift" because it's from the Swift standard library or more generally came from Swift. Does this generalize to things not in the standard library?

Reply:
In the second iteration I no longer need this approach because I now rely on attributes.